hrpsys_tools/scripts/rtmroswtf.sh

```diff
@@ -12,12 +43,30 @@ FILENAME_LOG_ALL=/tmp/rtmros_diagnosisinfo_all_`date +"%Y%m%d-%H%M%S"`.tgz
 env |grep ROS | tee -a ${FILENAME_LOG_COMMANDS}
 ifconfig | tee -a ${FILENAME_LOG_COMMANDS}
 rosrun rtshell rtls ${CORBA_HOSTNAME}:${CORBA_HOST_PORT}/ 2>&1 | tee -a ${FILENAME_LOG_COMMANDS}
-# Get packages' version. Ref. http://askubuntu.com/a/347563/24203
-dpkg -l | grep '^ii' | grep ros- | awk '{print $2 "\t" $3}' 2>&1 | tee -a ${FILENAME_LOG_COMMANDS}
+## Get packages' version. Ref. http://askubuntu.com/a/347563/24203
+## Get rosversion of the same packages
+res_dpkg=$(dpkg -l | grep '^ii' | grep ros- | awk '{print $2 " " $3}')
+IFS=$'\n'  # http://askubuntu.com/questions/344407/how-to-read-complete-line-in-for-loop-with-spaces
+for i in ${res_dpkg}
+do 
+  printf "%s $i " 2>&1 | tee -a ${FILENAME_LOG_COMMANDS}
+  p_underscore=$(echo ${i} | awk '{print $1}' | sed 's/ros-\([a-zA-Z]*\)-//' | tr '-' '_');
+  printf "%s ${p_underscore} " 2>&1 | tee -a ${FILENAME_LOG_COMMANDS}
+  rospackfind_result=$(rospack -q find ${p_underscore})
+  printf "%s ${rospackfind_result} " 2>&1 | tee -a ${FILENAME_LOG_COMMANDS}
+  echo "${rospackfind_result}" | grep -o '[^/]*$' | xargs rosversion 2>&1 | tee -a ${FILENAME_LOG_COMMANDS}
+  if [[ -z "$rospackfind_result" ]]; then
+    continue;
+  fi
+  # git status
```

Comment:
need to run git status on source package, not package found by `dpkg -s | grep ros...`, how about using `rospack list` and check if that directory have .git before running git status